src/wallet/wallet.cpp

```diff
@@ -1514,13 +1514,19 @@ int64_t CalculateMaximumSignedTxSize(const CTransaction &tx, const CWallet *wall
     return GetVirtualTransactionSize(txNew);
 }
 
+/* Prevout(32*4+4*4) + sequence(4*4) + scriptsig len(1*4) + scriptsig(22*4)
+* witness signature(1*71) + pubkey(1*33) + witness stack extra bytes(3)
+* converted to virtual size. This is used when estimating change spend size that
+* the wallet doesn't own or know how to sign for. */
+int EstimateP2SHP2WPKHInputSize() {
+    return (128+16+16+4+88+71+33+3)/4;
```

Comment:
Add whitespace to improve readability: `return (128 + 16 + 16 + 4 + 88 + 71 + 33 + 3) / 4;`